tests/sandbox/message_batches/create_message_batches/test_success.py

```diff
@@ -69,31 +80,41 @@ def test_201_message_batch_valid_nhs_number(nhsd_apim_proxy_url):
     .. include:: ../../partials/happy_path/test_201_message_batch_valid_nhs_number.rst
     """
     data = Generators.generate_valid_create_message_batch_body("sandbox")
```

Comment:
Again, optional. 

Is it worth moving this generation of data into a pre step. The consequence I think is that you'd have to spread it everywhere instead. Let me know your thoughts.

Reply:
The issue with doing that is that we have idempotency on the API thats related to the batch reference within this message - so it needs to be generated on a per test invocation basis or we'll end up getting back the deduped response.

Reply:
Given the data is different for each test I prefer as is rather than having to spread tbh